datahub-actions/src/datahub_actions/plugin/source/kafka/kafka_event_source.py

```diff
@@ -90,6 +91,10 @@ def build_entity_change_event(payload: GenericPayloadClass) -> EntityChangeEvent
 class KafkaEventSourceConfig(ConfigModel):
     connection: KafkaConsumerConnectionConfig = KafkaConsumerConnectionConfig()
     topic_routes: Optional[Dict[str, str]]
+    commit_delayed_enable: bool = False
```

Comment:
Is this technically "async committing?"

why not call this:

async_commit_enabled
async_commit_interval

Doesn't this capture what is happening more clearly?

Reply:
Renamed